src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -469,4 +481,15 @@ export default [
         },
         getDescription: () => {},
     },
+    {
+        isAnonymousAction: true,
+        textTranslateKey: 'reportActionContextMenu.menu',
+        icon: Expensicons.ThreeDots,
+        shouldShow: (type, reportAction, isArchivedRoom, betas, anchor, isChronosReport, reportID, isPinnedChat, isUnreadChat, isOffline, isMini) => isMini,
```

Comment:
Why are we showing / not showing only based on `isMini`?

Reply:
We only want to show the `ThreeDot` menu in the mini menu and not when we right click...